DNN Platform/Library/Application/Application.cs

```diff
@@ -230,7 +231,9 @@ public virtual Version Version
         {
             get
             {
-                return Assembly.GetExecutingAssembly().GetName().Version;
+                var assemblyLocation = Assembly.GetExecutingAssembly().Location;
+                var fileVersion = FileVersionInfo.GetVersionInfo(assemblyLocation).FileVersion;
+                return new Version(fileVersion);
```

Comment:
I realised that the version shown in Persona Bar is taken from `[AssemblyInfo]` so this will break with my changes (will always show `9.x.y (0)`). To fix I changed